### PR TITLE
Ingested MT-Summit 2007.

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/05/MTS-2007-TOC.htm for original.

* I have put the frontmatter for the main conference @ "Papers"
* I have combined the pdfs of the tutorials from Kevin Knight and Philipp Koehn into a single pdf

### CODE REVIEW

**Comment by davidstap** [comment]
I should fix the new bibkey stuff first, converted to draft for now.

**Comment by mjpost** [comment]
Yes, are you clear how to do this? Marcel has notes in #1431, I can try to assist if it's not clear.

**Comment by mjpost** [comment]
See [this comment](https://github.com/acl-org/acl-anthology/pull/1431#issuecomment-887829448).

**Comment by davidstap** [comment]
Thanks for the pointer, I think it’s clear. If not I’ll let you know :)

**Comment by mjpost** [comment]
Great, I'll wait till that's fixed so I can browse the preview

**Comment by mjpost** [comment]
Unfortunately you'll have to do the same for #1417, #1423, and #1428. It might be easiest to write a post-processing script to generate the keys, instead of reinvesting. Up to you of course.

**Comment by mbollmann** [comment]
There already is a script: `bin/write_bibkeys_to_xml.py`.

File: data/xml/2007.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,988 @@
...
+      <bibkey>vertan-2007-example</bibkey>
+    </paper>
+    <paper id="3">
+      <title>Context-based evaluation of <fixed-case>MT</fixed-case> systems: principles and tools</title>
+      <author><first>Maghi</first><last>King</last></author>
+      <author><first>Andrei</first><last>Popescu-Belis</last></author>
+      <author><first>Paula</first><last>Estrella</last></author>
+      <url hash="b4383f8a">2007.mtsummit-tutorials.3</url>
+      <attachment type="presentation" hash="91054497">2007.mtsummit-tutorials.3.Presentation.pdf</attachment>
+      <bibkey>king-etal-2007-context</bibkey>
```
Comment (mjpost) [inline data/xml/2007.mtsummit.xml:675]: There's an "outline and materials" attachment that got missed.

File: data/xml/2007.mtsummit.xml [hunk 2]
```diff
@@ -0,0 +1,988 @@
...
+    <paper id="7">
+      <title>On portability of resources for a quick ramp up of multilingual <fixed-case>MT</fixed-case> of patent claims</title>
+      <author><first>Svetlana</first><last>Sheremetyeva</last></author>
+      <url hash="f36dc245">2007.mtsummit-wpt.7</url>
+      <bibkey>sheremetyeva-2007-portability</bibkey>
+    </paper>
+  </volume>
+  <volume id="ucnlg" ingest-date="2021-07-30">
+    <meta>
+      <booktitle>Workshop on Using corpora for natural language generation</booktitle>
```
Comment (mjpost) [inline data/xml/2007.mtsummit.xml:762]: Proceedings of

File: data/xml/2007.mtsummit.xml [hunk 3]
```diff
@@ -0,0 +1,988 @@
...
+      <title>Content determination in <fixed-case>GRE</fixed-case>: evaluating the evaluator</title>
+      <author><first>Kees</first><last>van Deemter</last></author>
+      <author><first>Albert</first><last>Gatt</last></author>
+      <url hash="fe8cc183">2007.mtsummit-ucnlg.21</url>
+      <bibkey>van-deemter-gatt-2007-content</bibkey>
+    </paper>
+  </volume>
+  <volume id="aptme" ingest-date="2021-07-30">
+    <meta>
+      <booktitle>Workshop on Automatic procedures in MT evaluation</booktitle>
```
Comment (mjpost) [inline data/xml/2007.mtsummit.xml:920]: Proceedings of...

**Review by mjpost — COMMENTED** [review]
There's a workshop called "The Chinese Room Experiment" with just one paper that I think should be added.

File: data/xml/2007.mtsummit.xml [hunk 4]
```diff
@@ -0,0 +1,988 @@
...
+      <month>September 10</month>
+      <year>2007</year>
+      <editor><first>Bente</first><last>Maegaard</last></editor>
+    </meta>
+    <paper id="1">
+      <title>Statistical machine translation</title>
+      <author><first>Kevin</first><last>Knight</last></author>
+      <author><first>Philipp</first><last>Koehn</last></author>
+      <url hash="30aea1da">2007.mtsummit-tutorials.1</url>
+      <attachment type="presentation" hash="e49afcf4">2007.mtsummit-tutorials.1.Presentation.pdf</attachment>
```
[the remark this answers is not in the document]
Comment (mjpost) [reply data/xml/2007.mtsummit.xml:658]: Sorry, I missed your note about combining the PDFs.

**Comment by davidstap** [comment]
I did not include this workshop because there is not really a paper; it is only information about the workshop (basically Frontmatter). Should I still add this with John S. White and Florence Reeder (the chairs) as author?

**Comment by davidstap** [comment]
I have added the workshop anyway, this should be ready to merge.

**Comment by mjpost** [comment]
These should all be "Proceedings of the Workshop" (article missing from three of them)

**Review by mjpost — APPROVED** [review]
LGTM once the workshop titles are amended (add "the")!

**Pull request merged**